templates/organize/form/step2_application.html

```diff
@@ -0,0 +1,69 @@
+{% extends 'global/base.html' %}
+{% load i18n rev static %}
+
+{% block head %}
+{{ wizard.form.media }}
+{% endblock %}
+
+{% block extra_js %}
+    <script type="text/javascript" src="{% static_rev 'js/organize.js' %}"></script>
+{% endblock %}
+
+{% block content %}
+
+<div class="container organize">
+
+    <div class="row">
+        <div class="col-md-12">
+            {% include 'includes/_organize-progress.html' with active_step=3 %}
+        </div>
+    </div>
+
+    <div class="row">
+        <div class="col-md-12">
+            <div class="panel">
+                <h1>Application to organize a workshop</h1>
+                <form data-required-form action="" method="post">{% csrf_token %}
+                    {{ wizard.management_form }}
+
+                    <section class="form-section">
+                        <label for="{{ wizard.form.about_you.id_for_label }}" class="form-section-heading">Who are you?</label>
+                        <p class="form-section-subhead">Tell us a little bit about yourself. What’s your background and daily occupation?</p>
+                        {{ wizard.form.about_you }}
```

Comment:
Indeed they are, good spot. I've now matched this to the textareas in the later step we looked at together :)